src/controller/services.controller.js

```diff
@@ -0,0 +1,101 @@
+import serviceModel from "../model/services.model.js";
+import { response } from "express";
+
+export const servicesGet = async (req, res = response) => {
+  const { limite, desde } = req.query;
+  const query = { tp_status: "ACTIVE" };
+
+  const [total, services] = await Promise.all([
+    serviceModel.countDocuments(query),
+    serviceModel.find(query).skip(Number(desde)).limit(Number(limite)),
+  ]);
+
+  res.status(200).json({
+    total,
+    services,
+  });
+};
```

Comment:
Casi, pero necesito que uses `page` no desde, y para usar page, lo que vas a hacer va ser que vas a escribir `page` * `limit`
```suggestion
export const servicesGet = async (req, res = response) => {
  const { limite, page } = req.query;
  const query = { tp_status: "ACTIVE" };

  const [total, services] = await Promise.all([
    serviceModel.countDocuments(query),
    serviceModel.find(query).skip(Number(page) * Number(limit)).limit(Number(limite)),
  ]);

  res.status(200).json({
    total,
    services,
  });
};
```